presentation/src/main/java/com/mashup/alcoholfree/presentation/ui/component/model/SulSulButtonSize.kt

```diff
@@ -0,0 +1,19 @@
+package com.mashup.alcoholfree.presentation.ui.component.model
+
+import androidx.compose.ui.text.TextStyle
+import com.mashup.alcoholfree.presentation.ui.theme.SubTitle2
+import com.mashup.alcoholfree.presentation.ui.theme.SubTitle3
+import com.mashup.alcoholfree.presentation.ui.theme.SubTitle4
+
+enum class SulSulButtonSize(
+    val horizontalPadding: Int,
+    val verticalPadding: Int,
+    val imagePadding: Int,
+    val singleImagePadding: Int,
+    val radius: Int,
+    val textSize: TextStyle,
```

Comment:
```suggestion
    val textStyle: TextStyle,
```

size가 아니라 style인거같은데?!

Reply:
맞아여 형 저거 사실 어제 마지막에 발견하긴했는데 style로 바꾸는게 더 좋을것 같아요!